doc/kokkos-promotion.txt

```diff
@@ -175,6 +177,8 @@ Step 5: This step can be done on any SEMS machine (e.g. kokkos-dev). Actually, t
 
   5.4. Run checkin-test to push to trilinos using the CI build modules (gcc/4.9.3)
 
+     * This step is optional, the Github auto-tester is now the favored method for testing.
```

Comment:
More than optional -- we literally aren't allowed to do it.

Reply:
Good catch, @mhoemmen , this is true. We need to remove this entirely.